### PR TITLE
AUTHORS: update email

### PR DESCRIPTION
**Pull request by yehudasa**
Signed-off-by: Yehuda Sadeh <yehuda@redhat.com>

### CODE REVIEW

File: AUTHORS [hunk 1]
```diff
@@ -7,14 +7,14 @@ Sage Weil <sage@redhat.com>
...
-Core, RADOS   - Samuel Just <sam.just@inktank.com>
-RBD           - Josh Durgin <josh.durgin@inktank.com>
-RBD (kernel)  - Ilya Dryomov <ilya.dryomov@inktank.com>
-RGW           - Yehuda Sadeh <yehuda@inktank.com>
-CephFS        - Greg Farnum <greg@inktank.com>
+Core, RADOS   - Samuel Just <sjust@redhat.com>
+RBD           - Josh Durgin <jdurgin@redhat.com>
+RBD (kernel)  - Ilya Dryomov <idryomov@redhat.com>
+RGW           - Yehuda Sadeh <yehuda@redhat.com>
+CephFS        - Greg Farnum <gfarnum@redhat.com>
```
Comment (tchaikov) [inline AUTHORS]: @yehudasa can we take this chance to update Yan, Zheng's mail to "zyan@redhat.com"?

File: AUTHORS [hunk 2]
```diff
@@ -7,14 +7,14 @@ Sage Weil <sage@redhat.com>
...
                 Yan, Zheng <zheng.z.yan@intel.com>
-Deployment    - Alfredo Deza <alfredo.deza@inktank.com>
-Teuthology    - Zack Cerza <zack.cerza@inktank.com>
+Deployment    - Alfredo Deza <adeza@redhat.com>
+Teuthology    - Zack Cerza <zack@inktank.com>
```
Comment (tchaikov) [inline AUTHORS]: you meant "zack@redhat.com"?

**Comment by yehudasa** [comment]
@tchaikov yup, done

File: AUTHORS [hunk 3]
```diff
@@ -7,14 +7,14 @@ Sage Weil <sage@redhat.com>
...
-Core, RADOS   - Samuel Just <sam.just@inktank.com>
-RBD           - Josh Durgin <josh.durgin@inktank.com>
-RBD (kernel)  - Ilya Dryomov <ilya.dryomov@inktank.com>
-RGW           - Yehuda Sadeh <yehuda@inktank.com>
-CephFS        - Greg Farnum <greg@inktank.com>
-                Yan, Zheng <zheng.z.yan@intel.com>
-Deployment    - Alfredo Deza <alfredo.deza@inktank.com>
-Teuthology    - Zack Cerza <zack.cerza@inktank.com>
+Core, RADOS   - Samuel Just <sjust@redhat.com>
+RBD           - Josh Durgin <jdurgin@redhat.com>
```
Comment (theanalyst) [inline AUTHORS]: Won't RBD be handled by Jason Dillaman now?

**Comment by yehudasa** [comment]
@theanalyst @tchaikov updated

**Comment by theanalyst** [comment]
LGTM

**Comment by tchaikov** [comment]
lgtm.

**Pull request merged**